.ci/Jenkinsfile

```diff
@@ -63,7 +63,7 @@ pipeline {
               unstash 'source'
               dir("${BASE_DIR}"){
                 sh(label: 'make batch',
-                  script: """#!/bin/bash
+                  script: """#!/bin/bash -x
```

Comment:
Is this a leftover?
```suggestion
                  script: """#!/bin/bash
```

Reply:
Good catch, I needed to see what happened, since it didn't print anything but another make goal